### PR TITLE
Modified find_parameter_sources_and_assign_uncertainties to regenerate chem.inp

### PR DESCRIPTION
**Pull request by hwpang**
### Motivation or Problem
See #1941 for more details. I added a cell in find_parameter_sources_and_assign_uncertainties.ipynb to run the RMG job using input.py in /ipython/data/parse_source/ to regenerate a up-to-date chem.inp to avoid a break as described in #1939.

### Description of Changes
I added one cell ahead the uncertainty analysis cells to run the RMG job.

### Testing
I reran the whole ipynb and the results seem fine.

### CODE REVIEW

File: ipython/find_parameter_sources_and_assign_uncertainties.ipynb [hunk 1]
```diff
@@ -25,6 +25,31 @@
...
+    "!python ../rmg.py ./data/parse_source/input.py"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Perform parameter uncertainty assignment"
+   ]
+  },
```
Comment (xiaoruiDong) [inline ipython/find_parameter_sources_and_assign_uncertainties.ipynb:52]: I agree with the proposal for deleting those model jobs which may have incompatibility issues with the newest version. Just a minor comments. Can you add a comment/instruction to say this cell is only used when people want to try and learn this module with a toy model.

Comment (hwpang) [reply ipython/find_parameter_sources_and_assign_uncertainties.ipynb:53]: @xiaoruiDong Thanks for your review and comment! I've added the a short instruction in the markdown cell above this cell to inform the user when to use this cell.

**Comment by xiaoruiDong** [comment]
@amarkpayne related to #1941, I think I am fine with the implementation Hao-Wei did for this uncertainty notebook (removing the model and generating model in the notebook if necessary). If you also agree with the change, please help me merge this PR. Many thanks!

**Comment by hwpang** [comment]
@amarkpayne Thanks for your review and comments! I've added your suggested instruction on the cell for the user, and I've rebased with the new atom type error commit too.

**Pull request merged**